core/Tracker/Visit/VisitProperties.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tracker\Visit;
+
+/**
+ * Holds temporary data for tracking requests.
+ *
+ * RequestProcessors
+ */
+class VisitProperties
```

Comment:
Don't really understand we this class is needed and why it can't be just `Visitor` (see class further below). I'm sure there's a reason eg maybe BC? It would be really nice to have this in `Visitor` as it just sounds right